### ISSUE TITLE
Blockquote: spacing + full width alignment

### ISSUE THREAD

**Issue by beafialho** [WordPress/wporg-news-2021#141]
Large quotation mark style quotes should go full width. The spacing above them should be `40px` and below them `90px`

**Currently:**

<img width="1527" alt="Captura de ecrã 2021-12-20, às 16 02 12" src="https://user-images.githubusercontent.com/65220155/146797560-6fa9fd72-f916-403f-8e85-c9c538bd5191.png">

**Intended:**

<img width="715" alt="Captura de ecrã 2021-12-20, às 16 08 27" src="https://user-images.githubusercontent.com/65220155/146797619-86abee21-3e96-44eb-932f-071ab03e0149.png">

<img width="888" alt="Captura de ecrã 2021-12-20, às 16 08 03" src="https://user-images.githubusercontent.com/65220155/146798715-44a21ba0-6c01-403a-a540-739263e55c33.png">

**Comment by beafialho** [comment]
Can we add this to the Initial Launch milestone? Given that we'll get additional help, I think is worth tackling for the soft launch.

**Comment by iandunn** [comment]
I think it'd be better to keep the launch milestone for critical items, until all of those are done. Once they are, we can definitely move items from the post-launch milestone into pre-launch. Kind of like a "stretch goal".

If we put non-critical things in pre-launch, my worry is that folks will do those first because they're often easier, or because it's not clear what's critical and what isn't. If that happens then we may not have enough time to finish the critical items.

**Comment by kellychoffman** [comment]
Let's add this to the launch milestone, please. It's important we get these major design elements right as part of the launch of the redesign.

**Comment by iandunn** [comment]
> these major design elements

Were you referring to the blockquote spacing in this issue, or the category templates in #148 ?

Either way, I'd also like to launch with a complete design, but that's in tension with launching quickly. Adding more scope works for me, as long as you recognize that we'll have to extend the deadline, or launch without it, if we don't get enough contributors (or if Brook's Law kicks in).

**Comment by kellychoffman** [comment]
I disagree that this particular issue (the full width aspect especially) is blowing out the scope. This is a redesign and there's a level of design we need before we can ship. 

We'll do a final design review for issues that need to be included in the initial launch, but this is definitely one that needs to be in there.

**Comment by iandunn** [comment]
It's not just this issue, it's all the issues adding up.

**Comment by kellychoffman** [comment]
These meta discussions on the merit of each issue is becoming a bit discouraging and distracting from our overall goal. To move forward productively and efficiently, I propose two things: 

- I will bring in two additional designers with front end dev skills to support the overall goal of this project: ship a redesign (that does not sacrifice design quality)
- We have a level of trust with one another so that design + dev can both add issues to the corresponding milestones and that all of us will be reasonable when doing so.
